tests/jest/secrets/secrets_crud_operations.test.js

```diff
@@ -61,13 +61,7 @@ describe('Test secret crud operations', () => {
         expect(res.body).toMatchObject({
             namespace: testNamespace,
             secrets: {
-                pageInfo: {
-                    order: [],
-                    filter: [],
-                    limit: 0,
-                    offset: 0,
-                    total: 2
-                },
+                pageInfo: null,
```

Comment:
I'm not sure if the UI can handle this change. If it can, we should either TODO to fix this later OR we should consider updating the gRPC/protobuf API to reflect this change.

Reply:
It seems the frontend can handle this null. I still think we should remove it from our protobuf spec, but that can be a low priority TODO. Reduce the size of this PR by not also incorporating another protoc regeneration.